src/lit-element.ts

```diff
@@ -22,6 +22,28 @@ export {html, svg, TemplateResult, SVGTemplateResult} from 'lit-html/lit-html';
 import {supportsAdoptingStyleSheets, CSSResult} from './lib/css-tag.js';
 export * from './lib/css-tag.js';
 
+export interface LitElementStyles extends Array<CSSResult | CSSResult[] | LitElementStyles> {}
```

Comment:
Let's also allow this to be a single `CSSResult`. This way `static get styles` can be a single `CSSResult` and then any form of composition (sharing some "style modules" or some super class styles) is similar, e.g. `static get styles() { return [super.styles, sharedStyles]; }`

If we do this, then `flattenStyles` needs to handle the case where the input is a non-array and we'll obviously need to test that.

Reply:
Good suggestion, added it in :)